sdv/metadata/single_table.py

```diff
@@ -144,6 +144,13 @@ def _validate_column(self, column_name, sdtype, **kwargs):
             self._validate_datetime(column_name, **kwargs)
         elif sdtype == 'text':
             self._validate_text(column_name, **kwargs)
+        if 'pii' in kwargs:
+            pii_value = kwargs['pii']
+            if not isinstance(pii_value, bool):
+                raise InvalidMetadataError(
+                    f"Parameter 'pii' is set to an invalid attribute ({pii_value}). "
+                    'Expected a value of True or False.'
+                )
```

Comment:
extremely minor:
I think for consistency we could put this part in a `_validate_pii` method. You wouldn't have to change tests or anything for this.